Tutorials/Cache-Static-Assets/index.md

```diff
@@ -0,0 +1,99 @@
+---
+versionFrom: 9.3.0
+---
+
+# Cache static assets
+
+Example to cache static assets by file extension, but excluding Umbraco BackOffice assets.
+
+```csharp
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Net.Http.Headers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.Configuration.Models;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Core.Hosting;
+using Umbraco.Extensions;
+
```

Comment:
The code samples usually has a namespace declaration. While not mandatory it is good practice to add.

```suggestion

namespace My.Site.Composers;

```